apps/common/types/yearn.tsx

```diff
@@ -183,13 +183,3 @@ export type TYdaemonEarned = {
 	totalRealizedGainsUSD: number,
 	totalUnrealizedGainsUSD: number,
 }
-
-export type TYDaemonGaugeRewardsFeed = {
-	briber: TAddress;
-	gauge: TAddress;
-	rewardToken: TAddress;
-	amount: string;
-	txHash: string;
-	timestamp: string;
-	blockNumber: string;
```

Comment:
Another benefit of zod is to keep types up-to-date. For example, these 2 were coming as `number` but typed as `string`. If we get a different type, then the error will be logged in Sentry (hopefully soon) and we can change the schema to accommodate for that